src/content-layout/styles.scss

```diff
@@ -6,60 +6,101 @@
 @use '../internal/styles/tokens' as awsui;
 @use '../internal/generated/custom-css-properties/index.scss' as custom-props;
 
-/*
-Pass through the header and child content if not rendering in 
-visual refresh mode. The padding is still added to the header 
-in this scenario because otherwise the header and child 
-nodes will directly touch with no gap between them.
-*/
-.layout:not(.is-visual-refresh) {
-  display: contents;
-
-  > .background {
-    display: none;
-  }
-
-  > .header {
-    padding-block-end: awsui.$space-content-header-padding-bottom;
-  }
-}
-
-.layout.is-visual-refresh {
+.layout {
+  #{custom-props.$contentLayoutDefaultHorizontalPadding}: awsui.$space-layout-content-horizontal;
+  #{custom-props.$contentLayoutMaxContentWidth}: 0px;
+  #{custom-props.$contentLayoutMainGap}: 0px;
   display: grid;
-  grid-template-columns: minmax(0, 1fr);
-  grid-template-rows: auto awsui.$space-dark-header-overlap-distance 1fr;
+  grid-template-columns: 0 0 1fr minmax(0, var(#{custom-props.$contentLayoutMaxContentWidth})) 1fr 0 0;
+  grid-template-rows: var(#{custom-props.$contentLayoutMainGap}) min-content min-content auto awsui.$space-dark-header-overlap-distance 1fr;
   min-block-size: 100%;
 
+  @include styles.media-breakpoint-down(styles.$breakpoint-x-small) {
```

Comment:
So, we use Grid component inside the code which uses container queries for responsiveness and then we have plain media query here.

There is a chance these two will not agree on breakpoint values and there could be layout issues. What do you think about it?

Reply:
ContentLayout is supposed to be used with AppLayout and disabled content paddings. So its width would match the viewport width

Reply:
Media query does not react to drawers resize, for example. But the grid layout does. 

Do you see any possible issues when this happens?

Reply:
No, because the paddings apply only mobile view

Reply:
Okay, good here, but still a concerning pattern in general